Dockerfile

```diff
@@ -49,8 +52,10 @@ RUN git config --global --add safe.directory /usr/src/app
 
 # Run the setup script to identify the git commit hash and write it to a file.
```

Comment:
Probably outdated commend

Reply:
Not really, this is was `npm run identify` is doing